pypeit/core/coadd.py

```diff
@@ -2164,15 +2171,60 @@ def combspec(waves, fluxes, ivars, gpms, sn_smooth_npix,
     _fluxes = [np.float64(flux) for flux in fluxes]
     _ivars = [np.float64(ivar) for ivar in ivars]
 
+    # exposures that will not be coadded
+    nocoadd_exp = np.array([], dtype=int)
+
+    # check if there are exposures that are completely masked out, i.e., gpms = False for all pixels
+    masked_exp = np.where([np.all(gpm == False) for gpm in gpms])[0]
```

Comment:
Get rid of the where and use logical_not

bpm_exp = [np.all(np.logical_not(gpm) for gpm in gpms]